librFSM/CMakeLists.txt

```diff
@@ -79,6 +79,15 @@ install(TARGETS rFSM
         ARCHIVE DESTINATION lib
         PUBLIC_HEADER DESTINATION include)
 
+include(CMakePackageConfigHelpers)
+install(EXPORT rFSM
+        DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/rFSM)
+
+configure_package_config_file(${CMAKE_SOURCE_DIR}/cmake/modules/rFSMConfig.cmake.in
```

Comment:
Use `CMAKE_CURRENT_SOURCE_DIR` and `CMAKE_CURRENT_BINARY_DIR` in place of `CMAKE_SOURCE_DIR` and `CMAKE_BINARY_DIR`, to ensure that the project can be easily usable by using `FetchContent` and `add_subdirectory`, see https://github.com/robotology/ycm/issues/162 .

Reply:
Done in 65aa421